samtranslator/internal/schema_source/sam-docs.json

```diff
@@ -354,6 +354,7 @@
     },
     "sam-property-function-sns": {
       "FilterPolicy": "The filter policy JSON assigned to the subscription\\. For more information, see [GetSubscriptionAttributes](https://docs.aws.amazon.com/sns/latest/api/API_GetSubscriptionAttributes.html) in the Amazon Simple Notification Service API Reference\\.  \n*Type*: [SnsFilterPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sns-subscription.html#cfn-sns-subscription-filterpolicy)  \n*Required*: No  \n*AWS CloudFormation compatibility*: This property is passed directly to the [`FilterPolicy`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sns-subscription.html#cfn-sns-subscription-filterpolicy) property of an `AWS::SNS::Subscription` resource\\.",
+      "FilterPolicyScope": "This attribute lets you choose the filtering scope by using one of the following string value types:  \n+ `MessageAttributes` \\(default\\) \\- The filter is applied on the message attributes\\.\n+ `MessageBody` \\- The filter is applied on the message body\\.\n*Required*: No  \n*Type*: String  \n*Update requires*: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)",
```

Comment:
I assume this was added manually? `sam-docs.json` is generated automatically from https://github.com/awsdocs/aws-sam-developer-guide once updated. Instead you should leave out the assignment in `aws_serverless_function.py` and remove this. See also https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#making-schema-changes

Reply:
change reverted, thank you!

Reply:
Sorry I probably wasn't clear, what I wanted to say that it should just be:

```python
    FilterPolicyScope: Optional[PassThroughProp]
```

Without the ` = snseventproperties("FilterPolicyScope")` part (the assignment only adds the documentation to the schema), and without the change to `sam-docs.json`. Once done, run `make schema` to update the schema (CI will complain otherwise).

Reply:
added the property back without assignment, all ran `make schema`. Thank you!